packages/f-checkout/src/components/Checkout.vue

```diff
@@ -374,6 +389,21 @@ $line-height                              : 16px;
 $checkout-width                           : 596px;
 $checkout-padding                         : spacing(x5) 100px;
 
+.c-checkout-switchUserLink {
+  text-align: center;
+
+  a {
+    // TODO: check default link styles in PIE and update fozzie // (should be able to remove these styles then)
```

Comment:
Can this TODO be completed here? Because you copied it from f-registration component as I can see, which is not great